### PR TITLE
Skip update for energy modules when the bus isn't configured

### PR DESCRIPTION
**Pull request by khenderick**

### CODE REVIEW

File: src/power_bootloader.py [hunk 1]
```diff
@@ -299,6 +299,11 @@ def _get_from_ioc(power_store=INJECTED, power_communicator=INJECTED, power_seria
         return power_store, power_communicator, power_serial
 
     store, communicator, serial = _get_from_ioc()
+
```
Comment (svanscho) [inline src/power_bootloader.py:302]: perhaps split this over multiple lines with detailed log line? saves another round trip of checking

if store is None:
    logger.info('Energy store not found')
if communicator is None:
    logger.info('Energy bus is disabled')
   ....

**Review by svanscho — CHANGES_REQUESTED** [review]

File: src/power_bootloader.py (hunk 1 above)
Comment (khenderick) [reply src/power_bootloader.py:302]: As you can see here (https://github.com/openmotics/gateway/pull/517/files#diff-8a1a4eabd8ce905fc8d6a051b33944b4eba0b36fcd7d9732cdbb0185393cf77fR348) both will only be `None` when the bus is disabled.

Comment (svanscho) [reply src/power_bootloader.py:302]: I can't clearly see the path, but I would assume there is a possibility the db is corrupt/missing while the serial is configured. I am not saying it's likely, but possible.
Sidenote: is it planned somewhere to also migrate this energy power store to the general gateway.db?

Comment (khenderick) [reply src/power_bootloader.py:302]: There's a ticket for that

Comment (khenderick) [reply src/power_bootloader.py:302]: Updated

**Review by svanscho — APPROVED** [review]

**Pull request merged**